libcalico-go/lib/backend/k8s/conversion/workload_endpoint_default.go

```diff
@@ -253,6 +253,12 @@ func (wc defaultWorkloadEndpointConverter) podToDefaultWorkloadEndpoint(pod *kap
 		AllowSpoofedSourcePrefixes: requestedSourcePrefixes,
 	}
 
+	// multinet
+	multinet, ok := pod.Annotations["k8s.v1.cni.cncf.io/network-status"]
+	if ok {
+		wep.Annotations = make(map[string]string)
+		wep.ObjectMeta.Annotations["k8s.v1.cni.cncf.io/network-status"] = multinet
```

Comment:
We could probably copy the whole `Annotations` map here, that would prevent us to have to mention `network-status` in particular